lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -48,15 +48,20 @@ def static_control(*args, &block)
         options = args.extract_options!
         name = args.first
 
-        html = if block_given?
+        value = if block_given?
           capture(&block)
         else
           object.send(name)
         end
 
-        form_group_builder(name, options) do
-          content_tag(:p, html, { class: static_class }.merge(options[:id].present? ? { id: options[:id] } : {}))
-        end
+        static_options = {
+          value: value,
+          class: static_class,
+          readonly: true,
+          control_class: nil
```

Comment:
I'm curious why you assign `nil` to `control_class` here, and pass `static_class` via the `:class` key (two lines above)?

Reply:
Thanks for pointing this out, it was just an artifact of me walking through the problem and something I didn't realize wasn't necessary later.

A little after this as well these options are merged into the options that are passed into this method, so they will be overwritten. I think that with `control_class` that should be okay because we have the `class` option. The `readonly` option should be okay as well as it is necessary for the functionality. 

The only one I'm a bit iffy about is the `value` option. Now this can be set in the block, but I think it would probably be best to make it consistent with some of the other methods as well. I'll go ahead and make that change as well. If you have any objections let me know.

Reply:
I'm not sure I follow you completely. 

I think what you're doing with `value` is pretty good. Although it might be interesting to only assign to `options[:value]` if the block is given. That way if the user explicitly gives a `:value` option, it will work as they might expect. I think something like this:
```
options[:value] = yield if block_given?
```
I guess I would have thought it would be easiest to just add `form-control-plaintext` to anything that already happened to be in `options[:control_class]` and then let the logic of `text_field` do the rest. Something like:
```
options[:control_class] = [options[:control_class], static_class].compact.join(" ")
```
Make sure you check what I typed above, and that it makes sense. I hope you don't mind the nit-picking.